Julia/cp/isPowerOf2.jl

```diff
@@ -0,0 +1,26 @@
+#= Given a number N check if it is a power of 2 in O(1). We will be 
+using bitwise operator to get the calculation.=#
+
+## Function
+
+function isPowerOfTwo(n)
+    return (n & (n - 1) == 0)
+end
+
+## Input
+
+n = readline()
+n = parse(Int64, n)
+
+## Calling the function
+
+isPowerOfTwo(n)
```

Comment:
Just calling the function won't give any output. Please print the answer atleast.

Reply:
@atarax665 I have designed the function to be a boolean type, it will return true and false. I have even displayed a sample test case to show how it will function.